operator/.downstream_manifests

```diff
@@ -54,7 +54,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.17.0
+    controller-gen.kubebuilder.io/version: v0.18.0
   name: hooks.forklift.konveyor.io
```

Comment:
I think the makefile  generate-manifest is using the local developer kubectl. we should instead do that in a container. @mnecas what version of kubectl you run?